probe/probe.go

```diff
@@ -302,3 +334,362 @@ func probeNode(c *k8s.Client) {
+	renderOutputInTableWithNoBorders(data)
+	return true, nil
+}
+
+func getKubeArmorDeployments(c *k8s.Client) {
+	var data [][]string
+	_, err := boldWhite.Printf("Deployments : \n")
+	if err != nil {
+		color.Red(" Error while printing")
+	}
+
+	//relay deployment
+	relayDeployment, err := c.K8sClientset.AppsV1().Deployments("kube-system").Get(context.Background(), "kubearmor-relay", metav1.GetOptions{})
+	if err != nil {
+		return
+	}
+	//not updated replicas- what we need is desired replicas
+	desired1, ready1, available1 := relayDeployment.Status.UpdatedReplicas, relayDeployment.Status.ReadyReplicas, relayDeployment.Status.AvailableReplicas
+	data = append(data, []string{"\t KubeArmor-Relay: ", "Desired: " + strconv.Itoa(int(desired1)), "Ready: " + strconv.Itoa(int(ready1)), "Available: " + strconv.Itoa(int(available1))})
+
+	//host policy manager deployment
+	hostPolicyDeployment, err := c.K8sClientset.AppsV1().Deployments("kube-system").Get(context.Background(), "kubearmor-host-policy-manager", metav1.GetOptions{})
+	if err != nil {
+		return
+	}
+	//not updated replicas- what we need is desired replicas
+	desired2, ready2, available2 := hostPolicyDeployment.Status.UpdatedReplicas, hostPolicyDeployment.Status.ReadyReplicas, hostPolicyDeployment.Status.AvailableReplicas
+	data = append(data, []string{"\t KubeArmor-Host-Policy-Manager: ", "Desired: " + strconv.Itoa(int(desired2)), "Ready: " + strconv.Itoa(int(ready2)), "Available: " + strconv.Itoa(int(available2))})
+
+	//policy manager deployment
+	policyManagerDeployment, err := c.K8sClientset.AppsV1().Deployments("kube-system").Get(context.Background(), "kubearmor-policy-manager", metav1.GetOptions{})
+	if err != nil {
+		return
+	}
+	//not updated replicas- what we need is desired replicas
+	desired3, ready3, available3 := policyManagerDeployment.Status.UpdatedReplicas, policyManagerDeployment.Status.ReadyReplicas, policyManagerDeployment.Status.AvailableReplicas
+	data = append(data, []string{"\t KubeArmor-Policy-Manager: ", "Desired: " + strconv.Itoa(int(desired3)), "Ready: " + strconv.Itoa(int(ready3)), "Available: " + strconv.Itoa(int(available3))})
+
+	renderOutputInTableWithNoBorders(data)
+}
+
+func printContainers(c *k8s.Client) {
+	var data [][]string
+	kubearmorContainerCount, kubearmorImageVersion := getKubeArmorContainerCount(c)
+	relayContainerCount, relayImageVersion := getKubeArmorRelayContainerCountAndImageVersion(c)
+	_, err := boldWhite.Printf("Containers : \n")
+	if err != nil {
+		color.Red(" Error while printing")
+	}
+	if kubearmorContainerCount > 0 {
+		data = append(data, []string{"\t KubeArmor: ", "Running: " + strconv.Itoa(int(kubearmorContainerCount)), "image Version: " + kubearmorImageVersion})
+
+	}
+	if relayContainerCount > 0 {
+		data = append(data, []string{"\t KubeArmor-Relay: ", "Running: " + strconv.Itoa(int(relayContainerCount)), "image Version: " + relayImageVersion})
+	}
```

Comment:
Let's loop through like I suggested in the above comment and use a generic function to extract count and version.